guides/doc-Project_API/topics/ref_relating-api-error-messages-to-the-api-reference.adoc

```diff
@@ -0,0 +1,16 @@
+[id="relating-api-error-messages-to-the-api-reference"]
+= Relating API error messages to the API reference
+
+The API uses a RAILs format to indicate an error:
```

Comment:
What is "RAILs" format?

Also, the purpose of this file is not yet clear to me. Does this help debugging by telling me "if I get this error message via API, I can search for those errors in production.log"?

Reply:
It doesn't say anything about production.log, it speaks of the built-in API reference. But I'm not sure about the purpose either. Should we remove it?